Ginger/GingerCoreNET/UserCofig/UserProfile.cs

```diff
@@ -472,8 +472,10 @@ public void LoadRecentAppAgentMapping()
             {
                 //unserialize the current solution mapping saving
                 string existingSolMapping = RecentAppAgentsMapping.Where(x => x.Contains(WorkSpace.Instance.Solution.Name + "***") == true).FirstOrDefault();
+                Console.WriteLine("Agent Mapping:" + existingSolMapping);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this logging statement.](https://app.codacy.com/manual/Ginger-Automation/Ginger/pullRequest?prid=4707297)